_profiles/article.html

```diff
@@ -25,7 +25,7 @@
   connectivity: "Cable"
   location: "London_EC2"
   runs: 1
-  url: "https://next.wellcomecollection.org/articles/a-drop-in-the-ocean-daniel-regan"
```

Comment:
Could we potentially differentiate between WP & Prismic article (making prismic a default, and WP the exception)?

Reply:
Good idea, will do that in another PR